code/datums/emotes.dm

```diff
@@ -0,0 +1,169 @@
+#define EMOTE_VISIBLE 1
+#define EMOTE_AUDIBLE 2
+
+/datum/emote
+	var/key = "" //What calls the emote
+	var/key_third_person = "" //This will also call the emote
+	var/message = "" //Message displayed when emote is used
+	var/message_alien = "" //Message displayed if the user is a grown alien
```

Comment:
can this be `message_overrides = list()` instead and pass an associate list of string type to message?

Reply:
I am just porting /tg/ code at the moment, feel free to improve it later but this is out of scope